src/pudl/metadata/resources/eiaapi.py

```diff
@@ -4,9 +4,9 @@
 
 RESOURCE_METADATA: dict[str, dict[str, Any]] = {
     "core_eia__yearly_fuel_receipts_costs_aggs": {
-        "description": (
-            "Aggregated fuel receipts and costs from the EIA bulk electricity data."
-        ),
+        "description": {
+            "additional_summary_text": "aggregated fuel receipts and costs from the EIA bulk electricity data.",
+        },
```

Comment:
i think this one would be good to just say this comes from the eiaapi... honeslty not really sure why we decided to not just use the actual data source as the name here

Reply:
the data source name changed recently from eia_bulk_elec to eiaapi 🌈 ⭐